### PR TITLE
update doc for split region

### PR DESCRIPTION
**Pull request by crazycs520**
Signed-off-by: crazycs <crazycs520@gmail.com>

### What is changed, added or deleted? (Required)

As title said. The user complain the session variable of `SPLIT TABLE REGION` statement is hard to be attention beffore.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Comment by TomShawn** [comment]
@crazycs520 Please involve a technical review.

**Comment by crazycs520** [comment]
@TomShawn PTAL

File: sql-statements/sql-statement-split-region.md [hunk 1]
```diff
@@ -71,6 +71,13 @@ SPLIT TABLE table_name [INDEX index_name] BY (value_list) [, (value_list)] ...
 * `TOTAL_SPLIT_REGION`：表示新增预切分的 Region 数量。
 * `SCATTER_FINISH_RATIO`：表示新增预切分 Region 中，打散完成的比率。如 `1.0` 表示全部完成。`0.5`表示只有一半的 Region 已经打散完成，剩下的还在打散过程中。
 
+> **注意：**
+>
+> 以下 session variable 会影响 `SPLIT` 语句的行为，需要特别注意：
```
Comment (TomShawn) [inline sql-statements/sql-statement-split-region.md:76]:
```suggestion
> 以下会话变量会影响 `SPLIT` 语句的行为，需要特别注意：
```

File: sql-statements/sql-statement-split-region.md [hunk 2]
```diff
@@ -71,6 +71,13 @@ SPLIT TABLE table_name [INDEX index_name] BY (value_list) [, (value_list)] ...
...
+> 
+> * `tidb_wait_split_region_finish`: 由于打散 region 的时间可能比较长，主要由 PD 调度以及 TiKV 的负载情况所决定。这个变量用来设置在执行 `SPLIT REGION` 语句时，是否同步等待所有 region 都打散完成后再返回结果给客户端。默认 1 代表等待打散完成后再返回结果。0 代表不等待 Region 打散完成就返回。
```
Comment (TomShawn) [inline sql-statements/sql-statement-split-region.md:78]:
```suggestion
> * `tidb_wait_split_region_finish`：打散 Region 的时间可能较长，由 PD 调度以及 TiKV 的负载情况所决定。这个变量用来设置在执行 `SPLIT REGION` 语句时，是否同步等待所有 Region 都打散完成后再返回结果给客户端。默认 `1` 代表等待打散完成后再返回结果。`0` 代表不等待 Region 打散完成就返回结果。
```

File: sql-statements/sql-statement-split-region.md [hunk 3]
```diff
@@ -71,6 +71,13 @@ SPLIT TABLE table_name [INDEX index_name] BY (value_list) [, (value_list)] ...
...
+> * `tidb_wait_split_region_timeout`: 这个变量用来设置 `SPLIT REGION` 语句的执行超时时间，单位是秒，默认值是 300 秒，如果超时还未完成 Split ，就返回一个超时错误。
```
Comment (TomShawn) [inline sql-statements/sql-statement-split-region.md:79]:
```suggestion
> * `tidb_wait_split_region_timeout`：这个变量用来设置 `SPLIT REGION` 语句的执行超时时间，单位是秒，默认值是 300 秒，如果超时还未完成 `Split` 操作，就返回一个超时错误。
```

File: sql-statements/sql-statement-split-region.md [hunk 4]
```diff
@@ -179,6 +186,16 @@ SPLIT TABLE t INDEX idx2 BETWEEN ("2010-01-01 00:00:00") AND ("2020-01-01 00:00:
...
 
+如果希望按照天为间隔切分索引，示例如下：
+
+{{< copyable "sql" >}}
+
+```sql
+SPLIT TABLE t INDEX idx2 BETWEEN ("2020-06-01 00:00:00") AND ("2020-07-01 00:00:00") REGIONS 30;
+```
+
+该语句会将表 t 中 idx2 索引位于 2020 年 6 月份的数据按天为间隔切分成 30 个 region。
```
Comment (TomShawn) [inline sql-statements/sql-statement-split-region.md:197]:
```suggestion
该语句会将表 `t` 中 `idx2` 索引位于 2020 年 6 月份的数据按天为间隔切分成 30 个 Region。
```

File: sql-statements/sql-statement-split-region.md [hunk 5]
```diff
@@ -375,7 +392,10 @@ region4  [("c", "")                    , maxIndexValue               )
 
 > **注意：**
 >
-> `pre_split_regions` 必须小于等于 `shard_row_id_bits`。
+> 1. `pre_split_regions` 必须小于等于 `shard_row_id_bits`。
```
Comment (TomShawn) [inline sql-statements/sql-statement-split-region.md:395]: ditto

File: sql-statements/sql-statement-split-region.md [hunk 6]
```diff
@@ -375,7 +392,10 @@ region4  [("c", "")                    , maxIndexValue               )
...
+>
+> 2. 以下 global variable 会影响 `pre_split_regions` 的行为，需要特别注意：
+>     * `tidb_scatter_region`： 该变量用于控制建表完成后是否等待预切分和打散 Region 完成后再返回。如果建表后有大批量写入，需要设置该变量值为 1，表示等待所有 region 都切分和打散完成后再返回给客户端。否则未打散完成就进行写入会对写入性能影响有较大的影响。
```
Comment (TomShawn) [inline sql-statements/sql-statement-split-region.md:398]: ditto

File: sql-statements/sql-statement-split-region.md (hunk 5 above)
Comment (crazycs520) [inline sql-statements/sql-statement-split-region.md:395]:
```suggestion
> + `PRE_SPLIT_REGIONS` 必须小于等于 `SHARD_ROW_ID_BITS`。
```

File: sql-statements/sql-statement-split-region.md [hunk 7]
```diff
@@ -375,7 +392,10 @@ region4  [("c", "")                    , maxIndexValue               )
 
 > **注意：**
 >
-> `pre_split_regions` 必须小于等于 `shard_row_id_bits`。
+> + `PRE_SPLIT_REGIONS` 必须小于等于 `SHARD_ROW_ID_BITS`。
+>
+> 2. 以下 global variable 会影响 `pre_split_regions` 的行为，需要特别注意：
+>     * `tidb_scatter_region`： 该变量用于控制建表完成后是否等待预切分和打散 Region 完成后再返回。如果建表后有大批量写入，需要设置该变量值为 1，表示等待所有 region 都切分和打散完成后再返回给客户端。否则未打散完成就进行写入会对写入性能影响有较大的影响。
```
Comment (crazycs520) [inline sql-statements/sql-statement-split-region.md:398]:
```suggestion
>     * `tidb_scatter_region`：该变量用于控制建表完成后是否等待预切分和打散 Region 完成后再返回结果。如果建表后有大批量写入，需要设置该变量值为 `1`，表示等待所有 Region 都切分和打散完成后再返回结果给客户端。否则未打散完成就进行写入会对写入性能影响有较大的影响。
```

File: sql-statements/sql-statement-split-region.md [hunk 8]
```diff
@@ -375,7 +392,10 @@ region4  [("c", "")                    , maxIndexValue               )
 
 > **注意：**
 >
-> `pre_split_regions` 必须小于等于 `shard_row_id_bits`。
+> + `PRE_SPLIT_REGIONS` 必须小于等于 `SHARD_ROW_ID_BITS`。
+>
+> 2. 以下 global variable 会影响 `pre_split_regions` 的行为，需要特别注意：
```
Comment (TomShawn) [inline sql-statements/sql-statement-split-region.md:397]:
```suggestion
> + 以下全局变量会影响 `pre_split_regions` 的行为，需要特别注意：
```

**Pull request merged**